docs/DeveloperGuide.adoc

```diff
@@ -844,13 +882,34 @@ Use case ends.
 +
 Use case resumes at step 2.
 
-_{More to be added}_
+=== Use case: Search a person by tag
+*MSS*
+
+1. User requests to search and provides search params
+2. Addr	essBook shows a list of person fitting the given criteria
++
+Use case ends.
+
```

Comment:
What happened to extensions, such as the case where the user tries to find invalid tags/ tags that are not found in the address book?